composer.json

```diff
@@ -18,7 +18,8 @@
         "php-http/multipart-stream-builder": "^1.1",
         "psr/cache": "^1.0",
         "psr/http-client-implementation": "^1.0",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "php-http/mock-client": "^1.3"
```

Comment:
Why is this in the require block and not `require-dev`?

Reply:
I have added these dependencies in order to make tests. I have mock the client instance so every request made in test section does not call Bitbucket directly but returns a static response saved in a json file